pkg/util/runtime/runtime.go

```diff
@@ -36,7 +36,7 @@ type stackTracer interface {
 
 // replace the standard glog error logger, with a logrus one
 func init() {
-	logrus.SetFormatter(&joonix.FluentdFormatter{})
+	logrus.SetFormatter(joonix.NewFormatter())
```

Comment:
Do the fields match exactly?

Reply:
the fluentd format only uses three field: "time", "severity", "message". The issue [https://github.com/joonix/log/issues/2](https://github.com/joonix/log/issues/2) sets the default JSONFormatter with a FieldMap as a alternative.